src/utils.jl

```diff
@@ -44,19 +44,83 @@ vcat_arrays_vector(arrays_vector::AbstractVector{Param}) = vcat(Knet.cat1d.(arra
 """
     reset_minibatch_train!(nlp::AbstractKnetNLPModel)
 
-Select a new training minibatch for `nlp`.
-Typically used before a new evaluation of the loss function/gradient.
+Select the first training minibatch for `nlp`.
```

Comment:
Spell "mini batch" the same way everywhere.

Reply:
yes will update them